src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -379,7 +379,7 @@ static Gfx* D_808679A4[14][2] = {
     { gMilkBarDoorDL, gMilkBarDoorDL },
     { gMusicBoxHouseDoorDL, gMusicBoxHouseDoorDL },
     { gPiratesFortressDoorDL, gPiratesFortressDoorDL },
-    { object_mkk_DL_000310, object_mkk_DL_000310 },
+    { gWhiteBoeEyesDL, gWhiteBoeEyesDL },
```

Comment:
wat. Door?

Reply:
Ahh, that looks like a mistake. They should be the DLs from kinsta2, not mkk.

Reply:
ikr??? no idea

Reply:
> Ahh, that looks like a mistake. They should be the DLs from kinsta2, not mkk.

ohh ic so do i just leave em like that

Reply:
If you wouldn't mind changing them to use the kinsta2 dls that would be better. Would need to change the header include to the kinsta2 object, not the mkk one.

Reply:
might take the opportunity to do the remaining door assets as well then but sure